### PR TITLE
Week1_Exercise

### PR DESCRIPTION
**Pull request by HarshitPatel95**
Please review and let me know what you think.

### CODE REVIEW

File: Week-1/Exercise1.2/index.html [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+      href="https://fonts.googleapis.com/css?family=Lato"
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="https://maxst.icons8.com/vue-static/landings/line-awesome/line-awesome/1.3.0/css/line-awesome.min.css"
+    />
+  </head>
+  <body>
+    <main>
```
Comment (ShashankSuresh) [inline Week-1/Exercise1.2/index.html]: can we have a parent class?

File: Week-1/Exercise1.2/css/style.css [hunk 2]
```diff
@@ -0,0 +1,150 @@
+html {
+  height: 100%;
+}
+
+body {
+  display: flex;
+  flex-direction: column;
+  font-family: Lato, sans-serif;
```
Comment (ShashankSuresh) [inline Week-1/Exercise1.2/css/style.css]: Lets have common classes to all HTML elements using the `*` css selector 

```
* {
font-family: Lato, sans-serif;
}
```

File: Week-1/Exercise1.2/css/style.css [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+  flex: auto;
+}
+
+aside {
+  flex: 0 0 40px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  align-items: center;
+  background-color: #e3dfdf;
```
Comment (ShashankSuresh) [inline Week-1/Exercise1.2/css/style.css]: Lets have these in css variables

File: Week-1/Exercise1.2/css/style.css [hunk 4]
```diff
@@ -0,0 +1,150 @@
+html {
```
Comment (ShashankSuresh) [inline Week-1/Exercise1.2/css/style.css:5]: Lets have parents included to all children and grand-children

File: Week-1/Exercise1.1/readme.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
+# Exercise1.1
```
Comment (ShashankSuresh) [inline Week-1/Exercise1.1/readme.md:1]: Looks good. 

Can we have information about DNS - how it works, its purpose, caching, CDN etc ?
Can we also talk a bit more about what the response from the server looks like and how that is converted to DOM (HTML DOM + CSS DOM)

**Review by ShashankSuresh — CHANGES_REQUESTED** [review]

File: Week-1/Exercise1.2/css/style.css [hunk 6]
```diff
@@ -2,6 +2,11 @@ html {
   height: 100%;
 }
 
+:root {
+  --white: #ffffff;
```
Comment (ShashankSuresh) [inline Week-1/Exercise1.2/css/style.css:10]: Good job :)

File: Week-1/Exercise1.2/css/style.css [hunk 7]
```diff
@@ -25,126 +30,126 @@ aside {
...
   background-color: #ededed;
 }
 
-footer {
+.footer-content {
   display: flex;
   flex: 0 0 120px;
-  color: #fff;
+  color: var(--white);
   background-color: #5981bd;
```
Comment (ShashankSuresh) [inline Week-1/Exercise1.2/css/style.css]: Please add this to css variables.

File: Week-1/Exercise1.2/css/style.css [hunk 8]
```diff
@@ -25,126 +30,126 @@ aside {
...
+.main-content .content-section .list-content ul li p .sync-link {
+  border: 1px solid var(--black);
   font-size: 0.8em;
   padding: 2px;
   color: blue;
 }
 
-li:nth-child(2n) {
+.main-content .content-section .list-content ul li:nth-child(2n) {
   background-color: #ededed;
```
Comment (ShashankSuresh) [inline Week-1/Exercise1.2/css/style.css]: Please add this to css variables.

File: Week-1/Exercise1.2/index.html (hunk 1 above)
Comment (HarshitPatel95) [reply Week-1/Exercise1.2/index.html]: I have already done this change in my last commit with message name "feedback changes"

File: Week-1/Exercise1.2/css/style.css (hunk 4 above)
Comment (HarshitPatel95) [reply Week-1/Exercise1.2/css/style.css:5]: I have already done this change in my last commit with message name "feedback changes"

**Review by ShashankSuresh — CHANGES_REQUESTED** [review]

**Review by ShashankSuresh — APPROVED** [review]
Good job :)

**Pull request merged**